proxy-manager/Dockerfile

```diff
@@ -1,46 +1,41 @@
-ARG BUILD_FROM=ghcr.io/hassio-addons/base/amd64:10.0.1
+ARG BUILD_FROM=ghcr.io/hassio-addons/base/amd64:11.0.1
```

Comment:
The latest version of the NGinx Proxy Manager cannot run correctly on Alpine. The upstream project moved away from it.

Reply:
Bummer. But it looks to work flawless with my instance. Do you have any pointer on the reported Alpine incompatibility?
If that is the case, we can drop this PR because it is 90% Alpine based.

BTW, sorry if I'm wasting your time with these questions and PR. It was a quick fix for myself but tried to publish it so others can get the fix too.

Reply:
>  Do you have any pointer on the reported Alpine incompatibility?

There are quite a lot of pointers in the issue trackers upstream.

My currently plan/idea is to make a quite the breaking change, upgrading it to be Debian based (to match upstream) and replace MySQL with SQLite (as that is now possible nowadays). This removes the need for using MySQL on lower-end devices and makes backups more reliable as well.

Reply:
I see upstream is using Debian and the reasons may be explained here https://github.com/NginxProxyManager/nginx-proxy-manager/issues/1141#issuecomment-870243651
In the meantime I will leave the repo up for others to use, because really does seem to work well.

Your plan sounds good! +1
Feel free to close this PR. Thanks.